model-optimizer/mo/utils/custom_replacement_config.py

```diff
@@ -297,12 +299,12 @@ def update_custom_replacement_attributes(self, graph: Graph):
                 log.debug("Node {} doesn't have output edges. Consider it output".format(node_name))
                 output_tensors.add((generate_pattern_for_node(graph, pattern, node_name), 0))
 
-        if not self.has('inputs'):
+        if not self.has('inputs') or len(self._replacement_desc['inputs']) == 0:
```

Comment:
Why did you make this change and the change in line 307?

Reply:
We have cases when we have config files without inputs and outputs data. In the case MO will add values.
After validation we got config with empty input output array. So we need to add correct condition for the case.

Reply:
Empty input and output arrays are valid values. Instead of modifying the core code you should modify the validator